setup.py

```diff
@@ -1,4 +1,12 @@
 from setuptools import setup
 
+# version.py is created when building with SCons.
+# Otherwise, we create it ourselves.
+version = "0.1.0"
```

Comment:
I think the version information should stay in `setup.cfg`, so there's just one place that it lives. You can get it out easily:

```python
from distutils.dist import Distribution
d = Distribution()
d.parse_config_files()
version = d.get_version()
```

Reply:
By the way, "let's fix that in sconsUtils" is an acceptable response to this.